### DISCUSSION TITLE
IVF Index without Product Quantisation??

### DISCUSSION THREAD

**Discussion by rationalga** [lancedb/lancedb#759] [General]
Hi, May I know If we there is an option of creating IVF index without PQ like IVF_FLAT in Faiss? If not can we achieve the same effect by setting the num_sub_vectors = dimension_of_vector or is it a bad idea?? Thank you

**Comment by eddyxu** [comment]
@rationalga we dont have IVF_FLAT yet. 

I dont think we can do num_sub_vectors = dimension_of_vector, it does add quite some overhead to needlessly train PQ and reconstruct from PQ. Also it still compress float32 to uint8 even num_sub_vectors = dimension.

**Reply by rationalga** [reply]
oh got it, thank you.
